xbmc/platform/darwin/osx/powermanagement/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 set(SOURCES CocoaPowerSyscall.cpp)
-
 set(HEADERS CocoaPowerSyscall.h)
 
+if(CORE_PLATFORM_NAME_LC STREQUAL osx)
```

Comment:
we don't want to add Cocoa stuff when building for headless -> move to windowing

Reply:
This file is osx only, so need to check for osx.
As @FernetMenta already said, this should be moved to windowing as there is no need to control the display if nothing is shown.